src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -705,25 +705,25 @@ XamlObjectDefinition topLevelControl
 
 		private void BuildSingleTimeInitializer(IIndentedStringBuilder writer, string propertyType, string propertyName, Action propertyBodyBuilder, bool isStatic = true)
 		{
+			// The property type may be partially qualified, try resolving it through FindType
+			var propertySymbol = FindType(propertyType);
```

Comment:
Observation using `Windows.UI.Xaml.DataTemplate` as example input:
- `publicPropertyType `was not fully qualified actual, value was `DataTemplate` now `Windows.UI.Xaml.DataTemplate`
- `propertyType` was fully qualified is and remains `Windows.UI.Xaml.DataTemplate`
- 🐛 Qualification was happening after `publicPropertyType` was assigned by mutating `propertyType`

Thoughts:
- If we are going to mutate the `propertyType` parameter let's do it first.